theme/Shared/_Show.cshtml

```diff
@@ -83,7 +83,8 @@
 
                         if (showDates.Any(x => x > DateTime.Now) && Model.Get("box-office-open", false))
                         {
-                            var boxOfficeText = Model.GetString("box-office-text", "Book tickets");
+                            var modelText = Model.GetString("box-office-text", "Book tickets");
```

Comment:
I'm a little surprised that the fallback text "Book tickets" doesn't work here, that's what it's for! but oh well